spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.togglz</groupId>
+    <artifactId>togglz-project</artifactId>
+    <version>2.3.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>togglz-spring-boot-starter</artifactId>
+  <name>Togglz - Spring Boot Starter</name>
+  <description>Togglz - Spring Boot Starter</description>
+
+  <properties>
+    <hamcrest.version>1.3</hamcrest.version>
+    <hibernate-validator.version>5.2.2.Final</hibernate-validator.version>
+    <spring-boot.version>1.3.2.RELEASE</spring-boot.version>
+    <spring-security.version>4.0.3.RELEASE</spring-security.version>
```

Comment:
The parent module is not boot, but Togglz. Also note dependencies are optional so I don't see issue here.
In theory the actual dependnecy will be provided by Boot itself. I just need them to compile the code.